federation/pkg/federation-controller/replicaset/replicasetcontroller_test.go

```diff
@@ -180,6 +175,81 @@ func TestReplicaSetController(t *testing.T) {
 	assert.Equal(t, rs.Status.AvailableReplicas, rs1.Status.AvailableReplicas+rs2.Status.AvailableReplicas)
 }
 
+func TestReplicaSetControllerPuppets(t *testing.T) {
+
+	replicaSetReviewDelay = 10 * time.Millisecond
```

Comment:
It is needed to increase controller responsiveness. These are global variables. Discussed offline.